JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -443,7 +443,8 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
           addSingularFootnote <- TRUE
       }
       if (addSingularFootnote) {
-        model[["footnote"]] <- gettextf("%s Some confidence intervals could not be computed because out of the bootstrapped matrices none were invertible. ",
+        model[["footnote"]] <- gettextf("%s Some confidence intervals could not be computed because 
+                                        out of the bootstrapped matrices non were invertible. ",
```

Comment:
> Some confidence intervals could not be computed because out of the bootstrapped matrices non were invertible.

How about:

Some confidence intervals could not be computed because none of the bootstrapped covariance matrices were invertible.